### ISSUE TITLE
cannot open output file ObjToTGAConverter: Is a directory

### ISSUE THREAD

**Issue by DuendeInexistente** [Panzerschrek/Chasm-Reverse#39]
Trying to compile in ubuntu gives me this error. I bypassed it by disabling the compilation of tools in cmakecache.txt.

```
/usr/bin/ld: cannot open output file ObjToTGAConverter: Is a directory
collect2: error: ld returned 1 exit status
CMakeFiles/ObjToTGAConverter.dir/build.make:172: recipe for target 'ObjToTGAConverter' failed
make[2]: *** [ObjToTGAConverter] Error 1
CMakeFiles/Makefile2:67: recipe for target 'CMakeFiles/ObjToTGAConverter.dir/all' failed
make[1]: *** [CMakeFiles/ObjToTGAConverter.dir/all] Error 2
Makefile:83: recipe for target 'all' failed
make: *** [all] Error 2
```

**Comment by Panzerschrek** [comment]
Looks, like Cmake binary directory is also source directory.

**Comment by jopadan** [comment]
Fixed in #52 please close
